src/main/java/com/jaspersoft/jasperserver/jaxrs/client/apiadapters/authority/organizations/SingleOrganizationAdapter.java

```diff
@@ -145,4 +141,12 @@ public void run() {
                 new DefaultErrorHandler()
         );
     }
-}
+    // Deprecated method, use organization(organizationID).create() methods
+@Deprecated
```

Comment:
Please use @deprecated in javadoc (with suggestion) like here: http://www.atetric.com/atetric/javadoc/net.sf.jasperreports/jasperreports/5.6.0/src-html/net/sf/jasperreports/engine/JRExporterParameter.html